src/main/java/twistedgate/immersiveposts/client/ClientProxy.java

```diff
@@ -14,16 +14,12 @@
 /**
  * @author TwistedGate
  */
-public class ClientProxy extends CommonProxy{
+public class ClientProxy extends CommonProxy {
 	@Override
 	public void completed(){
 		ManualHelper.addConfigGetter(str -> {
-			switch(str){
-				case "maxTrussLength":{
-					return IPOConfig.MAIN.maxTrussLength.get();
-				}
-				default:
-					break;
+			if (str.equals("maxTrussLength")) {
+				return IPOConfig.MAIN.maxTrussLength.get();
```

Comment:
Future proofing, incase i want to add something else.

If i went the `if-statement` route this will just look disgusting.